locals.tf

```diff
@@ -84,8 +84,12 @@ locals {
   service_account_amp_ingest_name = format("%s-%s", module.aws_eks.cluster_id, "amp-ingest")
   service_account_amp_query_name  = format("%s-%s", module.aws_eks.cluster_id, "amp-query")
 
-  # Configuration for managing add-ons via GitOps.
-  gitops_add_on_config = {
-    awsForFluentBit = var.aws_for_fluentbit_enable ? module.aws_for_fluent_bit[0].gitops_config : null
+  # Configuration for managing add-ons via ArgoCD.
+  argocd_add_on_config = {
+    agones                    = var.agones_enable ? module.agones[0].argocd_gitops_config : {}
+    awsForFluentBit           = var.aws_for_fluentbit_enable ? module.aws_for_fluent_bit[0].argocd_gitops_config : null
+    awsLoadBalancerController = var.aws_lb_ingress_controller_enable ? module.aws_load_balancer_controller[0].argocd_gitops_config : null
+    certManager               = var.cert_manager_enable ? module.cert_manager[0].argocd_gitops_config : null
+    clusterAutoscaler         = var.cluster_autoscaler_enable ? module.cluster_autoscaler[0].argocd_gitops_config : null
```

Comment:
Should we make these variables naming convention as per terraform for consistency?  small case with underscores
`awsForFluentBit` to `aws_for_fluentbit`

Reply:
my thinking is that this object gets passed into Argo via the values map. So the variable consistency should likely align with K8s yaml? I.E. camel case?

Reply:
Got it! 
We can deal with this when we create a Helm module for this